release/windows/choco/tools/chocolateyinstall.ps1

```diff
@@ -2,7 +2,7 @@
 
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 $url        = '{{DownloadUrl}}'
-$url64      = 'https://download.microsoft.com/download/d/4/4/d4403a51-2ab7-4ea8-b850-d2710c5e1323/sqlcmd_0.8.1-1.msi'
+$url64      = 'https://github.com/microsoft/go-sqlcmd/releases/download/v1.3.0/sqlcmd-x64_1.3.0-1.msi'
 
```

Comment:
changing from the downloads site to github seems like a big deal. Do you know why we were using downloads before, and if there's any approval needed to stop using it?

And what is the value of `$url` here?

Reply:
I checked the download site for the previous version [here](https://community.chocolatey.org/packages/sqlcmd#files) and noticed that it is pointing to github. Thought of updating the files in our repo first to keep them in sync with choco. The choco command to push the package doesn't really rely on these files.